### DISCUSSION TITLE
Stuck on "Generating SSL certificate" for domain and the domain is working too

### DISCUSSION THREAD

**Discussion by originpulses** [vercel/community#5363] [Help]
### Summary

I have a Squarespace domain (originpulses.com) and I have added the following records to it:

@ | A | 76.76.21.21
www | CNAME | cname.vercel-dns.com

It has been stuck on "Generating SSL certificate" plus when I visit my domain as it shows: ERR_SSL_PROTOCOL_ERROR and fails to load.

On Vercel it shows that is setup but it isn't:
![Capture](https://github.com/vercel/community/assets/64883936/f31760b7-8e9d-48a6-b42f-d6c195f9f052)

### Example

originpulses.com

### Steps to Reproduce

What is wrong with my configuration, so frustrating as there is no response to the latest convos relating to this as well

**Comment by z4nr34l** [comment]
Please check if your DNS provider is not using any tunneling solutions like Cloudflare's proxy. Such solutions are blocking SSL generation.
